src/Http/Http.Abstractions/src/StatusCodes.cs

```diff
@@ -4,10 +4,13 @@
 namespace Microsoft.AspNetCore.Http;
 
 /// <summary>
-/// A collection of constants for HTTP status codes.
-///
-/// Status Codes listed at http://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml
+/// A collection of constants for
+/// <see href="http://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml" >HTTP status codes</see >.
 /// </summary>
+/// <remarks>
+/// Descriptions for status codes are available from
+/// <see cref="M:Microsoft.AspNetCore.WebUtilitiesReasonPhrases.GetReasonPhrase(Int32)" />.
+/// </remarks>
 public static class StatusCodes
 {
     /// <summary>
```

Comment:
Do you plan to include the description in each constant's `<summary>` like @Tratcher suggested in https://github.com/dotnet/aspnetcore/issues/38833#issuecomment-987033883? That seems like it could be helpful.

Reply:
It would be more helpful is to refer to the documentation of [HttpStatusCode](https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode) because 1DR > NIH.  But then, why not borrow the values defined in the system library?  I do not get that rejection at all.

Reply:
What's 1DR?

API cross references should be limited to APIs that will be directly used together. StatusCodes and HttpStatusCodes won't be used together. It's better for each of them to reference something authoritative like an RFC.

Reply:
1DR is the one definition rule.  According to this rule, [StatusCodes](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.http.statuscodes), defined a dependent library, is secondary to [HttpStatusCode](https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode), defined in the base library, and should import the values defined there rather than rolling its own (NIH).  It does not imply that [HttpStatusCode](https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode) should be used alongside [StatusCodes](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.http.statuscodes); just the opposite, except in the definition of the latter (or where you need a true `enum` for some reason, which may also happen). And the documentation of the dependent library, again by 1DR, should reference the documentation of the base library, except for bogus values that are not defined there.